gslib/utils/acl_helper.py

```diff
@@ -187,40 +187,44 @@ def _YieldMatchingEntries(self, current_acl):
             entry.entity.lower() == self.public_entity_all_auth_users.lower()):
         yield entry
 
-  def _AddEntry(self, current_acl, entry_class):
-    """Adds an entry to current_acl."""
+  def GetEntity(self):
+    """Gets an appropriate entity string for an ACL grant."""
     if self.scope_type == 'UserById':
-      entry = entry_class(entityId=self.identifier,
-                          role=self.perm,
-                          entity=self.user_entity_prefix + self.identifier)
+      return self.user_entity_prefix + self.identifier
     elif self.scope_type == 'GroupById':
-      entry = entry_class(entityId=self.identifier,
-                          role=self.perm,
-                          entity=self.group_entity_prefix + self.identifier)
+      return self.group_entity_prefix + self.identifier
     elif self.scope_type == 'Project':
-      entry = entry_class(entityId=self.identifier,
-                          role=self.perm,
-                          entity=self.project_entity_prefix + self.identifier)
+      return self.project_entity_prefix + self.identifier
     elif self.scope_type == 'UserByEmail':
-      entry = entry_class(email=self.identifier,
-                          role=self.perm,
-                          entity=self.user_entity_prefix + self.identifier)
+      return self.user_entity_prefix + self.identifier
```

Comment:
Duplicate `return` lines. Maybe `or` statements are in order?

Reply:
Sounds good!